apps/studio/pages/project/[ref]/database/backups/restore-to-new-project.tsx

```diff
@@ -0,0 +1,510 @@
+    debounce((value: string) => checkPasswordStrength(value), 300)
+  ).current
+
+  async function checkPasswordStrength(value: string) {
+    const { message, strength } = await passwordStrength(value)
+    setPasswordStrengthScore(strength)
+    setPasswordStrengthMessage(message)
+  }
+
+  function generateStrongPassword() {
+    const password = generator.generate({
+      length: 16,
+      numbers: true,
+      uppercase: true,
+    })
+
+    form.setValue('password', password)
+    delayedCheckPasswordStrength(password)
+  }
+
+  if (isLoading) {
+    return <GenericSkeletonLoader />
+  }
+
+  if (!canReadPhysicalBackups) {
+    return <NoPermission resourceText="view backups" />
+  }
+
+  if (!canTriggerPhysicalBackups) {
+    return <NoPermission resourceText="restore backups" />
+  }
+
+  if (!IS_PG15_OR_ABOVE) {
+    return (
+      <Admonition
+        type="default"
+        title="Restore to new project is not available for this database version"
+        description={
+          <>
+            Restore to new project is only available for Postgres 15 and above.
+            <br /> Go to{' '}
+            <Link className="underline" href={`/project/${project?.ref}/settings/infrastructure`}>
+              infrastructure settings
+            </Link>{' '}
+            to upgrade your database version.
+          </>
+        }
+      />
+    )
+  }
+
+  if (!PHYSICAL_BACKUPS_ENABLED) {
+    return (
+      <Admonition
+        type="default"
+        title="Restore to new project requires physical backups"
+        description={
+          <>
+            Enable physical backups to restore to a new project.
+            <br /> Find out more about how backups work at supabase{' '}
+            <Link
+              target="_blank"
+              className="underline"
+              href="https://supabase.com/docs/guides/platform/backups"
+            >
+              in our docs
+            </Link>
+          </>
+        }
+      />
+    )
+  }
+
+  if (plan === 'free') {
+    return (
+      <UpgradeToPro
+        buttonText="Upgrade"
+        primaryText="Restore to a new project requires a pro plan or above."
+        secondaryText={
+          'To restore to a new project, you need to upgrade to a Pro plan and have physical backups enabled.'
+        }
+      />
+    )
+  }
+
+  if (isError) {
+    return <AlertError error={error} subject="Failed to retrieve backups" />
+  }
+
+  if (!isActiveHealthy) {
+    return (
+      <Admonition
+        type="default"
+        title="Restore to new project is not available while project is offline"
+        description={<>Your project needs to be online to restore your database to a new project</>}
+      />
+    )
+  }
+
+  if (cloneStatus?.status === 'FAILED') {
+    return (
+      <Admonition
+        type="destructive"
+        title="Failed to restore to new project"
+        description={
+          <>
+            The new project failed to be created.
```

Comment:
do we get any other information about why it failed? i do think we should explain a little instead of just directly pointing them to support